galsim/base.py

```diff
@@ -2892,6 +2892,119 @@ def __str__(self):
 _galsim.SBSpergel.__setstate__ = lambda self, state: 1
 _galsim.SBSpergel.__repr__ = lambda self: \
         'galsim._galsim.SBSpergel(%r, %r, %r, %r, %r)'%self.__getinitargs__()
+    
+
+class InclinedExponential(GSObject):
+    """A class describing an inclined exponential profile.
+
+    The Inclined Exponential surface brightness profile is characterized by three properties: its
+    inclination angle 'i' (where 0 = face-on and pi/2 = edge-on), its scale radius, and its scale
+    height. The 3D light distribution function is:
+    
+    I(R,z) = I_0 / (2h_s) * sech^2 (z/h_s) * exp(-R/R_s)
+    
+    where z is the distance along the minor axis, R is the radial distance from the minor axis,
+    R_s is the scale radius of the disk, h_s is the scale height of the disk, and I_0 is the central
+    surface brightness of the face-on disk.
+    
+    In this implementation, the profile is inclined along the y-axis. This means that it will likely
+    need to be rotated in most circumstances.
+    
+    At present, this profile is not enabled for photon-shooting.
+    
+    Initialization
+    --------------
+
+    @param i                The inclination angle, i
```

Comment:
Not `i`.  Parameters should be at least somewhat self-documenting via their names.  `inclination` would probably be appropriate.

Also, should mention here that it is an `Angle`.